src/Symfony/Component/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -47,10 +47,10 @@ protected function describeInputArgument(InputArgument $argument, array $options
     {
         $this->write(
             '#### `'.($argument->getName() ?: '<none>')."`\n\n"
-            .($argument->getDescription() ? preg_replace('/\s*[\r\n]\s*/', "\n", $argument->getDescription())."\n\n" : '')
```

Comment:
Offtopic, but CS related, isn't there a rule available to keep the `.` on the end of the previous line?

Reply:
not currently. we have one to normalize size of single-line whitespace around `.`, but when it's multi-line, no changes.

rule could be added, but core team needs to focus on 8.4 compatibility first.